lib/lambda/submit/submitSplitSPA.ts

```diff
@@ -0,0 +1,117 @@
+import { response } from "libs/handler-lib";
+import { APIGatewayEvent } from "aws-lambda";
+import { getPackage } from "libs/api/package";
+import { produceMessage } from "libs/api/kafka";
+import { ItemResult } from "shared-types/opensearch/main";
+import { getNextSplitSPAId } from "./splitSPAId";
+import { z } from "zod";
+
+const sendSubmitSplitSPAMessage = async (currentPackage: ItemResult) => {
+  const topicName = process.env.topicName as string;
+  if (!topicName) {
+    throw new Error("Topic name is not defined");
+  }
+  try {
+    const newId = await getNextSplitSPAId(currentPackage._id);
+    if (!newId) {
+      throw new Error("Error getting next Split SPA Id");
+    }
+
+    // ID and changeMade are excluded; the rest of the object has to be spread into the new package
+    const {
+      id: _id,
+      changeMade: _changeMade,
+      origin: _origin,
+      ...remainingFields
+    } = currentPackage._source;
+
+    await produceMessage(
+      topicName,
+      newId,
+      JSON.stringify({
+        id: newId,
+        idToBeUpdated: currentPackage._id,
+        ...remainingFields,
+        origin: "OneMAC",
+        changeMade: "OneMAC Admin has added a package to OneMAC.",
+        changeReason: "Per request from CMS, this package was added to OneMAC.",
+        isAdminChange: true,
+        adminChangeType: "split-spa",
+      }),
+    );
+
+    return response({
+      statusCode: 200,
+      body: { message: `New Medicaid Split SPA ${newId} has been created.` },
+    });
+  } catch (err) {
+    console.log("Error creating Split SPA: ", err);
+  }
```

Comment:
Should we continue bubbling up the error? Will parent `try`/`catch` capture this?